integrations/terraform-cloud/main.py

```diff
@@ -165,8 +164,21 @@ async def resync_state_versions(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
         yield enriched_state_versions_batch
 
 
+@ocean.on_resync()
+async def on_create_webhook_resync(kind) -> ASYNC_GENERATOR_RESYNC_TYPE:
```

Comment:
The `create_workspace_webhook` method is not an ASYNC_GENERATOR. After checking, it doesn't return anything

Look through [here](https://github.com/port-labs/ocean/blob/main/port_ocean/core/ocean_types.py) to determine the best return type

Reply:
okay, I'm choosing RAW_RESULT from the options